contracts/asset-proxy/test/dydx_bridge.ts

```diff
@@ -122,21 +119,50 @@ blockchainTests.resets('DydxBridge unit tests', env => {
                 expectedOperateActionEvents.push({
                     actionType: action.actionType as number,
                     accountId: action.accountId,
-                    amountSign: action.actionType === BridgeActionType.Deposit ? true : false,
+                    amountSign: action.actionType === DydxBridgeActionType.Deposit ? true : false,
                     amountDenomination: weiDenomination,
                     amountRef: deltaAmountRef,
                     amountValue: action.conversionRateDenominator.gt(0)
-                        ? amount.times(action.conversionRateNumerator).div(action.conversionRateDenominator)
+                        ? amount
+                              .times(action.conversionRateNumerator)
+                              .dividedToIntegerBy(action.conversionRateDenominator)
                         : amount,
                     primaryMarketId: marketId,
                     secondaryMarketId: constants.ZERO_AMOUNT,
-                    otherAddress: action.actionType === BridgeActionType.Deposit ? from : to,
+                    otherAddress: action.actionType === DydxBridgeActionType.Deposit ? from : to,
                     otherAccountId: constants.ZERO_AMOUNT,
                     data: '0x',
                 });
             }
             verifyEventsFromLogs(txReceipt.logs, expectedOperateActionEvents, TestDydxBridgeEvents.OperateAction);
         };
+        it('succeeds when calling with zero amount', async () => {
```

Comment:
Yeah that's true. Although I feel like for local unit tests it could be good to include the edge cases, in case we change how the external code works.